src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj

```diff
@@ -50,6 +51,9 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
We do not support Json 11.x.x at the moment. Please start an offline thread regarding your requirements here

Reply:
Can you set the proper nuget packages for this pull request?

Reply:
Nuget package references have been set globally [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/AzSdk.reference.props)
Hope that helps

Reply:
Updated

Reply:
@krajniak could you tell us why you need these specific Newtonsoft references
Also, if you are trying to use a different auth mechanism to interact with the REST service, we do not support it at the moment.

Reply:
I don't, should I change it to not-specific version?

Reply:
Ok, I've reverted the *.csproj changes, and only change the version of  Microsoft.Azure.Management.ResourceManager.Fluent to 1.16.0

Reply:
> @krajniak please use the above link to sign CLA before we even look at your contribution.

Done